MewsSystems/mews-flutter
# feat!: [UX-135] Update input style

#### Summary

- refactored input fields
- tokens instead of the old colors
- **[BREAKING CHANGE]** - signature change, the `secondaryCaption` is now `helperMessage` for the sake of consistency with the design and web version
- updated story

#### Testing steps

- check the Input Field story
- fiddle with the knobs to test all exported options
- check if the dark theme view is correct

#### Follow-up issues

- update Number Input Field

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
